emgapianns/management/commands/create_study_summary.py

```diff
@@ -64,9 +60,12 @@ def handle(self, *args, **options):
         study_accession = options['accession']
         pipeline = options['pipeline']
         database = options['database']
+
+        if not options['rootpath']:
+            raise ValueError("rootpath (RESULTS_PRODUCTION_DIR setting) cannot by empty)")
+
```

Comment:
Should we have an `assert os.path.isdir(rootpath)` in here too, to prevent chaos from CLI typos?

Reply:
agree